Marlin/src/lcd/ultralcd.cpp

```diff
@@ -1082,17 +1082,16 @@ void kill_screen(const char* lcd_msg) {
           const int left = 5, right = 90, nozzle = 60;
         #endif
 
-          // Draw a representation of the nozzle
-          if (PAGE_CONTAINS(3, 16))  u8g2.drawBitmap(nozzle + 6, 4 - dir, 2, 12, PSTR(nozzle_bmp));
-          if (PAGE_CONTAINS(20, 20)) u8g2.drawBitmap(nozzle + 0, 20, 3, 1, PSTR(offset_bedline_bmp));
-
-          // Draw cw/ccw indicator and up/down arrows.
-          if (PAGE_CONTAINS(47,62)) {
-            u8g2.drawBitmap(left  + 0, 47, 3, 16, PSTR(rot_down));
-            u8g2.drawBitmap(right + 0, 47, 3, 16, PSTR(rot_up));
-            u8g2.drawBitmap(right + 20, 48 - dir, 2, 13, PSTR(up_arrow_bmp));
-            u8g2.drawBitmap(left  + 20, 49 - dir, 2, 13, PSTR(down_arrow_bmp));
-          }
+        // Draw a representation of the nozzle
+        if (PAGE_CONTAINS(3, 16))  u8g2.drawBitmap(nozzle + 6, 4 - dir, 2, 12, PSTR(nozzle_bmp));
```

Comment:
Is `PSTR()` needed here? I'm not sure what it does with pointers to things other than strings.

Reply:
Do we need a PPTR() ?   A Program Memory Based Pointer ???   My guess is PSTR handles arbitrary pointer just fine.   But the name PSTR implies the generated pointer is to a text string, which isn't true.

Reply:
If the u8g2 library expects these pointers to be `PROGMEM` then it already does the right thing, so `PSTR` is redundant. But if u8g2 expects normal SRAM pointers, maybe `PSTR` magically remaps them…? I'll wait and see whether `PSTR` was needed to make it work before assuming it's bad.

Reply:
It's the replacement for original constructs like
```
u8g.drawBitmapP(nozzle + 6, 4 - dir, 2, 12, nozzle_bmp);
```
Looks ok to me.

Reply:
Where nozzle_bmp is in PROGMEM.

Reply:
So… `PSTR` is not needed then. GTK!